Sources/Basic/Path.swift

```diff
@@ -205,6 +207,11 @@ public struct AbsolutePath: Hashable {
         return _impl.string
     }
 
+    /// File URL created from the normalized string representation of the path.
+    public var asURL: Foundation.URL {
```

Comment:
Let's make this as an extension so we can avoid importing Foundation in this file. We can put the extension in `misc.swift`.

Reply:
done